protocols/relay/tests/lib.rs

```diff
@@ -378,18 +377,18 @@ impl From<ping::Event> for RelayEvent {
     prelude = "libp2p_swarm::derive_prelude"
 )]
 struct Client {
-    relay: client::Client,
+    relay: relay::client::Behaviour,
     ping: ping::Behaviour,
 }
 
 #[derive(Debug)]
 enum ClientEvent {
-    Relay(client::Event),
+    Relay(relay::client::Event),
     Ping(ping::Event),
 }
 
-impl From<client::Event> for ClientEvent {
-    fn from(event: client::Event) -> Self {
+impl From<relay::client::Event> for ClientEvent {
+    fn from(event: relay::client::Event) -> Self {
         ClientEvent::Relay(event)
     }
 }
```

Comment:
If you are feeling like deleting a bit more code, we could turn on `event_process = true` (which is the default) and remove this. Can easily be done in a follow-up though.

Reply:
ehe yeah done. Also deleted the `ClientEvent` and `RelayEvent` that's what you were meaning right? And also removed from the examples.

Reply:
Yes!